src/test/java/de/uhd/ifi/se/decision/management/jira/extraction/tangledcommitdetection/TestTangledCommitDetection.java

```diff
@@ -0,0 +1,86 @@
+package de.uhd.ifi.se.decision.management.jira.extraction.tangledcommitdetection;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import de.uhd.ifi.se.decision.management.jira.extraction.Diff;
+import de.uhd.ifi.se.decision.management.jira.extraction.TangledCommitDetection;
+import de.uhd.ifi.se.decision.management.jira.extraction.gitclient.TestSetUpGit;
+import de.uhd.ifi.se.decision.management.jira.extraction.impl.DiffImpl;
+import de.uhd.ifi.se.decision.management.jira.extraction.impl.TangledCommitDetectionImpl;
+
+public class TestTangledCommitDetection extends TestSetUpGit {
```

Comment:
@lalilalalalu , some tests fail on Travis CI but they pass in my machine. Do you know why?